test/customer_test.rb

```diff
@@ -3,22 +3,26 @@
 require_relative '../lib/customer'
 
 class CustomerTest < MiniTest::Test
+  attr_reader :parent, :customer
+  def setup
+    sales_engine = SalesEngine.new
+    @parent = CustomerRepository.new('test/support/sample_customers.csv', sales_engine)
```

Comment:
Line is too long. [87/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.